client/app.vue

```diff
@@ -19,16 +19,22 @@ useEventBus(WebsocketPacketReceivedEvent).on(({ payload }) => {
     case 'user-forcefully-logged-out': {
       setTokens(null)
       setUser(null)
-      const redirect = getRedirectParam(route.path)
-      navigateTo(`/login?reason=${encodeURIComponent(payload.data.reason)}${redirect}`)
+      const redirect = currentRoute.value.query.redirect ?? route.path
+      navigateTo({
+        path: '/login',
+        query: { reason: payload.data.reason, redirect },
+      })
       break
     }
   }
 })
```

Comment:
might as well use the new composable

Reply:
Hmm, this code snippet handles multiple events, so it can remain like that I guess...

Reply:
you can do as you wish, that more a suggestion than anything